aws_quickstart/datadog_integration_autoscaling_policy.yaml

```
@@ -0,0 +1,65 @@
+# version: v<VERSION_PLACEHOLDER>
+AWSTemplateFormatVersion: 2010-09-09
+Description: Attach autoscaling SetDesiredCapacity policy to the existing Datadog AWS Integration IAM role
+Parameters:
+  IAMRoleName:
+    Description: Name of the existing IAM role for Datadog AWS integration
+    Type: String
```

```suggestion
    Type: String
    MinLength: 1
    MaxLength: 64
```
See [API Reference](https://docs.aws.amazon.com/IAM/latest/APIReference/API_Role.html#:~:text=RoleName).